src/components/AppViews.js

```diff
@@ -24,6 +24,7 @@ class AppViews extends Component {
             <Vote
               currentUser={currentUser} {...props}
               setCurrentUser={setCurrentUser}
+              updateCurrentUserVote={this.props.updateCurrentUserVote}
```

Comment:
Re-enable eslint on this file when you get a chance

Reply:
refactored from class to functional component - done!